lib/awful/hotkeys_popup/keys/tmux.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------
+--- tmux hotkeys for awful.hotkeys_widget
+--
+-- @autor nahsi nashi@airmail.cc
+-- @copyright 2017 nahsi
+-- @module awful.hotkeys_popup.keys.tmux
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local tmux_rule = { name = "tmux" }
```

Comment:
@blueyed  btw since you're using the same naming convention for you client names with tmux sessions, mb you will find this helprer useful :)
https://github.com/actionless/awesome_config/blob/devel/utils/tmux.lua